licenses.yaml

```diff
@@ -233,6 +233,30 @@ notice: |
 
 ---
 
+name: JavaBeans Activation Framework API JAR
+license_category: binary
+module: java-core
+license_name: Eclipse Distribution License 1.0
+version: 1.2.1
+copyright: Oracle and/or its affiliates.
+license_file_path: licenses/bin/jakarta.activation-api.EDL1
+libraries:
+  - jakarta.activation: jakarta.activation-api
+
+---
+
+name: Jakarta XML Bind API
+license_category: binary
+module: java-core
+license_name: Eclipse Distribution License 1.0
+version: 2.3.2
+copyright: Oracle and/or its affiliates.
+license_file_path: licenses/bin/jakarta.activation-api.EDL1
```

Comment:
Since the EDL1 license is used for both jakarta.activation-api and jakarta.xml.bind-api, I'll rename the license file to "jakarta.EDL1"